falcon/testing/helpers.py

```diff
@@ -40,7 +68,7 @@ def rand_string(min, max):
     int_gen = random.randint
     string_length = int_gen(min, max)
     return ''.join([chr(int_gen(ord(' '), ord('~')))
-                    for i in range(string_length)])
+                    for __ in range(string_length)])
```

Comment:
TBH, I changed this on a whim while reviewing this function. I've started following this recommendation in my work:

http://docs.python-guide.org/en/latest/writing/style/#create-an-ignored-variable